modules/l0/src/main/scala/org/elpaca_metagraph/l0/custom_routes/CustomRoutes.scala

```diff
@@ -42,6 +42,8 @@ case class CustomRoutes[F[_] : Async](calculatedStateService: CalculatedStateSer
           case "walletsholdingdag" => Ok(dataSources.get(WalletCreationHoldingDAG))
           case "freshwallets" => Ok(dataSources.get(FreshWallet))
           case "existingwallets" => Ok(dataSources.get(ExistingWallets))
+          case "inflowtransactions" => Ok(dataSources.get(InflowTransactions))
```

Comment:
This whole part looks like a lot of repetitive code. It should be simplified to return the response once and get the data source based on its name